### PR TITLE
Add secondary tabs for database query

### LINKED ISSUE #13621: QBE selenium tests broken since merge of #13342
The Selenium tests for QBE got broken since merge of #13342:

```
1) PmaSeleniumQueryByExampleTest::testQueryByExample

InvalidArgumentException: Element not found.

/home/travis/build/phpmyadmin/phpmyadmin/test/selenium/TestBase.php:390

/home/travis/build/phpmyadmin/phpmyadmin/test/selenium/PmaSeleniumQueryByExampleTest.php:69
```

PS: This was hard to catch before, I'm working on bringing Selenium test to pull requests in https://github.com/phpmyadmin/phpmyadmin/issues/13603

### PR DESCRIPTION
**Pull request by MauricioFauth**
### Description

Added secondary tabs for database query to fix QBE selenium tests.

Fixes #13621

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by mauriciofauth** [comment]
@ibennetch Since `db_qbe.php` wasn't removed by #13342 in `QA_4_8`, I thought it was better to just add a secondary tab that links to `db_qbe.php`.
I think we can remove `db_qbe.php` and the _visual builder_ as described by #13268 and [New UI for Multi-table query generator](https://github.com/phpmyadmin/phpmyadmin/wiki/GSoC_2017_ideas_list#new-ui-for-multi-table-query-generator) in `master`, if it's still desirable. What do you think?

**Comment by williamdes** [comment]
@mauriciofauth Is #14658 still valid ?

**Comment by ibennetch** [comment]
I'm torn on the solution to this. If we keep the old QBE, this is an excellent solution. However, the original plan was to completely remove QBE and replace it with the new interface.

I don't feel like I have enough information about how other people use the tool to make a decision. I'm happy with the replacement, for what that's worth.

File: db_qbe.php [hunk 1]
```diff
@@ -138,6 +139,23 @@
...
 $db_qbe = new Qbe($GLOBALS['db'], $savedSearchList, $savedSearch);
 
+$secondaryTabs = [
+    'multi' => [
+        'link' => 'db_multi_table_query.php',
+        'text' => __('Multi-table query'),
+    ],
+    'qbe' => [
+        'link' => 'db_qbe.php',
+        'text' => __('Query by example'),
```
Comment (mauriciofauth) [inline db_qbe.php]: @ibennetch I'm not sure about the tabs names: _Multi-table query_ and _Query by example_. I'm just followed the respective file names.

**Pull request merged**